### DISCUSSION TITLE
Is the ESP32-2432S024 functionally the same as the ESP32-2432S028R CYD?

### DISCUSSION THREAD

**Discussion by rocket59** [witnessmenow/ESP32-Cheap-Yellow-Display#35] [General]
There some ads up on Aliexpress for this board which looks similar save for a capacitive screen and USB C. Does anyone know if these can be used instead ?

**Comment by Tertiush** [comment]
Have a look here: https://community.home-assistant.io/t/openhasp-an-mqtt-driven-touchscreen-scene-controller/300853/415

**Reply by AnhTuWally** [reply]
> Have a look here: https://community.home-assistant.io/t/openhasp-an-mqtt-driven-touchscreen-scene-controller/300853/415

Would you please provide more information on why this link is useful?
